spec/meilisearch/client/snapshots_spec.rb

```diff
@@ -0,0 +1,13 @@
+# frozen_string_literal: true
+
+RSpec.describe 'MeiliSearch::Client - Snapshots' do
+  it 'creates a new snapshot' do
+    response = client.create_snapshot
```

Comment:
Be sure to test if the returned task type has the correct type: `snapshotCreation`.

Reply:
done